components/src/ChatBot/chatbot.review.tsx

```diff
@@ -0,0 +1,109 @@
+/**
+ * Copyright © 2018 Elastic Path Software Inc. All rights reserved.
+ *
+ * This is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this license. If not, see
+ *
+ *     https://www.gnu.org/licenses/
+ *
+ *
+ */
+
+import React from "react";
+import Amplify, { Interactions } from 'aws-amplify';
+import awsmobile from "../../../app/src/aws-exports";
```

Comment:
explicitly requiring this import will cause the project build to fail if the file doesn't exist and the user hasn't configured the chatbot